### ISSUE TITLE
Screen Reader Can't Access ledDelay and ledBuffers.

### ISSUE THREAD

**Issue by chigkim** [corrados/jamulus#499]
In clientdlg.cpp, the following items are set.
`ledDelay->setAccessibleName ( tr ( "Delay status LED indicator" ) );`
`ledBuffers->setAccessibleName ( tr ( "Buffers status LED indicator" ) );`

However, they never gets updated, so screen reader users cannot access actual information.
For delay, `CClientDlg::OnPingTimeResult` should update `ledDelay->setAccessibleDescription(tr("Green"));`
For buffer, `CClientDlg::OnTimerBuffersLED` should update `ledBuffers->setAccessibleDescription(tr("Green"));`

**Comment by corrados** [comment]
What about the bars? Are the values of the bars is used by the screen reader, i.e., if you change the skin to Normal or Compact skin?

**Comment by chigkim** [comment]
I'm actually using normal skin.

On Mac, VoiceOver just announces them as "Image" regardless skin.

On fancy skin, they appear as multiple images, but still no value.

**Comment by corrados** [comment]
I have just added the setAccessibleDescription to the Delay and Buffer LEDs: https://github.com/corrados/jamulus/commit/a708a53cd0229c3b7fb0a667c5ad07264f167612
Can you please review and test the new code?

**Comment by chigkim** [comment]
Thanks! Both read the colors!
